<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Account</class>
 <widget class="QDialog" name="Account">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>700</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Account</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: #3b5998</string>
  </property>
  <widget class="QFrame" name="frame">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>110</y>
     <width>700</width>
     <height>460</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QFrame{
background-color: rgb(83, 86, 173);
border: 5px solid white;
margin: 20% 0;}</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>160</y>
      <width>221</width>
      <height>231</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);
border-radius:15px;</string>
    </property>
    <property name="title">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>50</y>
      <width>181</width>
      <height>91</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>80</y>
     <width>351</width>
     <height>501</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QGroupBox{background-color: rgb(255, 255, 255);}
QLineEdit{
	
border: 2px solid #6b6b6b ;
  border-radius: 3px;
 padding: 10px 20px;
	background-color: rgb(255, 255, 255);}</string>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QPushButton" name="pushButton_1">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>150</y>
      <width>61</width>
      <height>42</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe UI Black</family>
      <pointsize>12</pointsize>
      <italic>false</italic>
      <fontweight>Black</fontweight>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
    width: 100px;
    background: #ee0979;
    font-weight: bold;
    color: white;
    border: 0 none;
    border-radius:20px;
    padding: 7px 5px;
	font: 900 12pt &quot;Segoe UI Black&quot;;
}


QPushButton::hover{
   border: 2px solid white;
    width: 100px;
    background: #ee0979;
    font-weight: bold;
    color: white;
     border-radius:20px;
    padding: 7px 5px;
	font: 900 12pt &quot;Segoe UI Black&quot;;
}
</string>
    </property>
    <property name="text">
     <string>1</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_name">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>40</y>
      <width>281</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>18</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(0, 0, 0);
border:none;</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>235</y>
      <width>61</width>
      <height>42</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe UI Black</family>
      <pointsize>12</pointsize>
      <italic>false</italic>
      <fontweight>Black</fontweight>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
    width: 100px;
    background: #ee0979;
    font-weight: bold;
    color: white;
    border: 0 none;
    border-radius:20px;
    padding: 7px 5px;
	font: 900 12pt &quot;Segoe UI Black&quot;;
}


QPushButton::hover{
   border: 2px solid white;
    width: 100px;
    background: #ee0979;
    font-weight: bold;
    color: white;
     border-radius:20px;
    padding: 7px 5px;
	font: 900 12pt &quot;Segoe UI Black&quot;;
}
</string>
    </property>
    <property name="text">
     <string>2</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_3">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>320</y>
      <width>61</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe UI Black</family>
      <pointsize>12</pointsize>
      <italic>false</italic>
      <fontweight>Black</fontweight>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
    width: 100px;
    background: #ee0979;
    font-weight: bold;
    color: white;
    border: 0 none;
    border-radius:20px;
    padding: 7px 5px;
	font: 900 12pt &quot;Segoe UI Black&quot;;
}


QPushButton::hover{
   border: 2px solid white;
    width: 100px;
    background: #ee0979;
    font-weight: bold;
    color: white;
     border-radius:20px;
    padding: 7px 5px;
	font: 900 12pt &quot;Segoe UI Black&quot;;
}
</string>
    </property>
    <property name="text">
     <string>3</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_data">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>161</y>
      <width>111</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);
font: 700 12pt &quot;Segoe UI&quot;;
text-decoration: underline;
color: rgb(0, 0, 0);</string>
    </property>
    <property name="text">
     <string>MY DATA</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_data_card">
    <property name="geometry">
     <rect>
      <x>24</x>
      <y>246</y>
      <width>221</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);
font: 700 12pt &quot;Segoe UI&quot;;
text-decoration: underline;
color: rgb(0, 0, 0);</string>
    </property>
    <property name="text">
     <string>INFORMATION CARD</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_payments">
    <property name="geometry">
     <rect>
      <x>13</x>
      <y>331</y>
      <width>241</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe UI</family>
      <pointsize>12</pointsize>
      <italic>false</italic>
      <bold>true</bold>
      <underline>true</underline>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);
font: 700 12pt &quot;Segoe UI&quot;;
text-decoration: underline;
color: rgb(0, 0, 0);</string>
    </property>
    <property name="text">
     <string>PAYMENTS/TRANSFERS</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
